### PR TITLE
Adds machine learning workshop post

### PR DESCRIPTION
**Pull request by claricen**

### CODE REVIEW

File: content/S2016/S16-event-machine-learning.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+JavaScript before!
+
+Space for the workshop is limited so make sure to [register soon](https://www.eventbrite.com/e/machine-learning-tech-workshop-tickets-25891397868)!
+
+Technical Workshop Series: This event is generously sponsored by 
+[BlackBerry](http://ca.blackberry.com/home.html), as part of our 
+Technical Workshop Series. We are hosting three such workshops 
+focused on teaching technical skills in partnership with BlackBerry. 
+To learn more about our Technical Workshop Series and partnership 
+with BlackBerry see our sponsors page.
```
Comment (ehashman) [inline content/S2016/S16-event-machine-learning.md]: Should have a link to the sponsors page.

**Comment by ehashman** [comment]
Otherwise LGTM.

**Comment by claricen** [comment]
What's the link for the sponsors page? I can't find it on the site...

**Comment by fboxwala** [comment]
http://wics.uwaterloo.ca/sponsorship/

^^actually this needs to be updated as well. We need to add the git workshop and this one to the ending blurb. Would you be able to open a PR for that afterwards as well? If not I'll do tonight.

**Comment by fboxwala** [comment]
Its linked via the FAQ because navbar.

**Comment by claricen** [comment]
Added link ^^

**Pull request merged**